Makefile

```diff
@@ -3,7 +3,7 @@ ORG             := cnabio
 BINDIR          := $(CURDIR)/bin
 GOFLAGS         :=
 GOBUILDTAGS     := osusergo
-LDFLAGS         := -w -s
+LDFLAGS         := -w -s -X github.com/cnabio/signy/pkg/docker.Tag=$(TAG)
```

Comment:
Are we actually setting the `$TAG` variable?
Running `make build`, the tag is empty, because it is not set:

```
$ go build  -tags 'osusergo' -ldflags '-w -s -X github.com/cnabio/signy/pkg/docker.Tag=' -o /Users/radu/projects/src/github.com/cnabio/signy/bin/signy github.com/cnabio/signy/cmd/...
```

Reply:
Given that there is no `version` command either, I'd rather have a separate issue tracking adding the correct build tag and a version command.

I'll open a separate issue about this.